api-test/fetch-user.js

```diff
@@ -0,0 +1,27 @@
+
+const cardContainer = document.getElementById('card-container');
+const fetchData = async()=>{
+  try{
+    const res = await fetch('https://jsonplaceholder.typicode.com/users')
```

Comment:
What link is this? You could have simply put a json file and used that

Reply:
Okay sir

Reply:
Sir I've replaced the link with json file.